### ISSUE TITLE
az pipelines run --parameters is throwing unrecognized arguments error

### ISSUE THREAD

**Issue by neoswallow** [Azure/azure-cli#21534]
**Describe the bug**
running `az pipelines run` with `--paramters` option throws **_unrecognized arguments: --parameters_** error.

**To Reproduce**
`az pipelines run --id 3 --parameters platform=ios --output table`
result in
`unrecognized arguments: --parameters platform=ios`

**Expected behavior**
It should run the pipeline of given id with parameter platform set to ios

**Environment summary**

Install Method: homebrew
CLI version: (result of az --version)
```
azure-cli                         2.34.1

core                              2.34.1
telemetry                          1.0.6

Extensions:
azure-devops                      0.22.0
interactive                        0.4.5

Dependencies:
msal                              1.16.0
azure-mgmt-resource               20.0.0

Python location '/opt/homebrew/Cellar/azure-cli/2.34.1/libexec/bin/python'
Extensions directory '/Users/*****/.azure/cliextensions'

Python (Darwin) 3.10.2 (main, Feb  2 2022, 05:51:25) [Clang 13.0.0 (clang-1300.0.29.3)]
```

OS version: macOS 12.2.1
Shell Type: zsh, fish

**Additional context**

Document website https://docs.microsoft.com/en-us/cli/azure/pipelines?view=azure-cli-latest#az-pipelines-run list this option but it simply doesn't work, at least on macOS version.

**Comment by yonzhan** [comment]
route to CXP team

**Comment by navba-MSFT** [comment]
Adding Service Attention label for the service team to look into this.

**Comment by kaito-ms** [comment]
The `--parameters` option is implemented in azure-devops 0.23.0.
Please upgrade your azure-devops extension.

https://github.com/Azure/azure-devops-cli-extension/releases/tag/20220131.1
> Pipeline client 6.0 added to support template parameters for pipelines runs (https://github.com/Azure/azure-devops-cli-extension/pull/1211)

**Comment by neoswallow** [comment]
@kaito-ms
I've been using ubuntu version (via docker) meanwhile.
It looks like it worked with ubuntu version because it installed v0.25.0 when my local version was still at v0.22.0.
Now that I updated devops extension to the latest (v0.25.0), --parameters works now.
Thanks.

I do see that table output is somehow broken with the new version for the pipeline.
I'm guessing I should create an issue on devops extension repo.
